src/job/job.spec.js

```diff
@@ -167,4 +167,33 @@ describe("job", () => {
       ).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".runSequence() :", () => {
+    afterEach(async () => {
+      await Promise.all(
+        xo.objects.all[config.vmIdXoTest].snapshots.map(id =>
+          xo
+            .call("vm.delete", { id, delete_disks: true })
+            .catch(error => console.error(error))
+        )
+      );
+    });
+
+    it("runs a job", async () => {
+      const id = await xo.createTempJob(defaultJob);
+      const snapshotsLength =
+        xo.objects.all[config.vmIdXoTest].snapshots.length;
+
+      await waitObjectState(xo, config.vmIdXoTest, ({ snapshots }) =>
```

Comment:
We don't did a particular operation which change the VM state. So we shouldn't wait for a new particular state.